color.go

```diff
@@ -0,0 +1,54 @@
+package termui
+
+import "github.com/fatih/color"
+
+// These mirror their equivalent functions in ui.go
+// Unfortunately since Print is already variadic,
+// we can't have an elegant "...color.Attribute" argument and have to use a slice
+
+func (u *UI) PrintColor(colors []color.Attribute, args ...interface{}) (int, error) {
+	colorString := color.New(colors...).SprintFunc()(args...)
+	return u.Print(colorString)
+}
+
+func (u *UI) PrintfColor(colors []color.Attribute, format string, args ...interface{}) (int, error) {
+	colorString := color.New(colors...).SprintfFunc()(format, args...)
+	return u.Print(colorString)
+}
+
+func (u *UI) PrintlnColor(colors []color.Attribute, args ...interface{}) (int, error) {
+	colorString := color.New(colors...).SprintlnFunc()(args...)
+	return u.Print(colorString)
+}
+
+// Helper functions for simple color printing
+
+func (u *UI) RedPrintln(args ...interface{}) (int, error) {
+	colors := []color.Attribute{color.FgRed}
+	return u.PrintlnColor(colors, args...)
```

Comment:
Not sure why you need an intermediate.
```golang
return u.PrintlnColor([]color.Attribute{color.FgRed}, args...)
```
Is it to make it match the color + bold variants better? But those don't look long enough to require intermediate variables either…